frontend/public/models/yaml-templates.ts

```diff
@@ -403,14 +431,26 @@ apiVersion: v1
 kind: Pod
 metadata:
   name: example
+  annotations:
+    image.openshift.io/triggers: |-
```

Comment:
I don't think this will work for an individual pod since `image` is immutable after the pod is created.